.github/workflows/pre-submit.actions.yml

```diff
@@ -319,3 +320,79 @@ jobs:
           directory: "internal/builders/generic"
           # NOTE: compile-builder explicitly set to false.
           compile-builder: false
+
+  generate-attestations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create test inputs
```

Comment:
you could store these results in a file committed to the repo

Reply:
done! i'll add more malicious layouts and predicate validations in these folders anyway.